src/test/java/com/eighthlight/RouterTest.java

```diff
@@ -64,6 +81,15 @@ public void createsUriRouteWithPostMethodHandler(){
         Assert.assertTrue(testRouter.getRoutes().get("/route").containsKey("POST"));
     }
 
+    @Test
+    public void createUriRouteWithOptionsMethodHandler(){
+        Router testRouter = new Router();
+        testRouter.options("/route", new HeadHandler());
+
+        Assert.assertTrue(testRouter.getRoutes().containsKey("/route"));
+        Assert.assertTrue(testRouter.getRoutes().get("/route").containsKey("OPTIONS"));
```

Comment:
Something to keep in mind here when creating assertions. Generally, I try to avoid writing assertions that evaluate a bit of code and check whether the result of that code is `true` or `false`. The reason being is that if this test were to fail in the future, the failure message would say something like `Expected true, but got false`, which really isn't super helpful when trying to debug something. Especially when its been weeks/months/years later and you have to remember what this part of the code does exactly. So, I might write this assertion by asserting on the expected return value:

```suggestion
        Assert.assertEquals(testRouter.getRoutes().get("/route"), "OPTIONS");
```

This failure would likely say `Expected "OPTIONS", but got "HEAD"` or something like that, which is a bit more descriptive.